includes/tax-functions.php

```diff
@@ -259,15 +259,18 @@ function edd_get_formatted_tax_rate( $country = false, $state = false ) {
  *
  * @since 1.3.3
  * @since 3.0 Renamed $state parameter to $region.
+ *            Added $fallback parameter.
  *
  * @param float  $amount  Amount.
  * @param string $country Country. Default base country.
  * @param string $region  Region. Default base region.
+ * @param boolean $fallback Fall back to the current Customer's address information
+ *                          or server $_POST data. Default true.
```

Comment:
@ashleyfae Can we update this (and the `@param` on `edd_get_tax_rate()`) to mention the final fallback being the store's address settings?

Reply:
Good idea - updated!